forge.config.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Configuration file for Electron Forge
+ */
+module.exports = {
+  packagerConfig: {
+    asar: true,
+    // TODO: Honeycomb icon (hexagon on docs)
+    icon: "assets/icons/icon",
+  },
+  makers: [
+    {
+      // zip files
+      name: "@electron-forge/maker-zip",
+    },
+    {
+      // Linux Distribution
+      name: "@electron-forge/maker-deb",
+      config: {
+        options: {
+          // TODO: Honeycomb icon (hexagon on docs)
+          // icon: "assets/icons/icon.png",
+        },
+      },
+    },
+    {
+      // Mac Distribution
+      name: "@electron-forge/maker-dmg",
+      config: {
+        // TODO: Honeycomb icon (hexagon on docs)
+        icon: "assets/icons/icon.icns",
+        overwrite: true,
+      },
+      platforms: ["darwin"],
+    },
+    {
+      // Windows Distribution
+      name: "@electron-forge/maker-squirrel",
+      config: {
+        // TODO: Honeycomb icon (hexagon on docs)
+        // TODO: This will break when we merge with main

```

Comment:
Is this an issue for existing honeycomb users or is it just the icon so they'll get like a missing image?

Reply:
The URL is point to `main` and we're changing the path to that file so we'll have to update it when this PR gets into `main` (it's going into a different branch right now)

Reply:
Oh wait I can totally update the URL here for now, no one's going to be packaging based on this branch